blob/service.go

```diff
@@ -304,3 +259,25 @@ func (s *Service) getBlobs(
 	}
 	return SharesToBlobs(namespacedShares.Flatten())
 }
+
+// toAppShares converts node's raw shares to the app shares, skipping padding
+func toAppShares(shrs ...share.Share) ([]shares.Share, error) {
+	appShrs := make([]shares.Share, 0)
```

Comment:
do you want to allocate up to cap here too?

Reply:
It's okay to over-allocate slices usually. The reason is that you still do one allocation compared to N times depending on the len of `shrs`. As you know, each time you don't have enough space in the slice, the new 2x one is allocated.

Reply:
True.

Reply:
So do you want to alloc to cap here or no?

Reply:
yes, updated. Thank you